src/operations.py

```diff
@@ -157,16 +157,16 @@ def center(_item, xdata, ydata, center_maximum):
     return new_xdata, ydata, True, False
 
 
-def shift_vertically(item, xdata, ydata, left_scale, right_scale, datadict):
+def shift_vertically(item, xdata, ydata, left_scale, right_scale, items):
```

Comment:
In the main class, we now changed the attribute previously known as `datadict` to `data`, in this function it's called `items`. 

I'm happy with a rename, as `datadict` is maybe not ideal (the idea was that it was a dictionary that contains all the data), but perhaps we should stick to a more consistent naming throughout.

Reply:
items in this case is a list of all items. Before this function would just query all items from the datadict for each item this operation was performed on. This refactor moves this query out to the caller function (window.py/shift_vertically)